### ISSUE TITLE
INVITES: Give a preview of the topic in invite emails?

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1540]
@kevincrepin In discussion with @ilmartyrk today, we were thinking that showing a bit of a preview of the topic, e.g. topic image, topic intro, (maybe even a bit of the topic text but see what you think) could be good to include in the invite emails as well.

Like with [this issue](https://github.com/citizenos/citizenos-fe/issues/1376) the idea would be to show people what it is they're being invited to, so that they understand more why they should register, when they are asked to do so. (And are more motivated to go through the process of doing so, as a result of knowing a bit of what they'll find in the topic).

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @ssin1901 Put this one into 'in prep', as seems it should be done together with this: https://github.com/citizenos/citizenos-fe/issues/1376

**Comment by kevincrepin** [comment]
I think the invite e-mail actually holds quite a lot of info already, but we could add the image (if there is one) and the topic intro text (or in the future current phase part of text).

![Screenshot 2024-09-27 at 12 13 24](https://github.com/user-attachments/assets/cd4550b2-0e16-4c32-94f5-a650d4965e66)

@ilmartyrk https://www.figma.com/design/gOJ4i00538e0qaP6IKIPVE/Platform-2.0---Design?node-id=14730-141792&t=3hHy3U6W9ohEF2Lu-1

**Comment by BeccaMelhuish** [comment]
@kevincrepin Looks great! I'd probably not include the 'topic group' though 

And I guess we should do the same for groups? :)

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @ssin1901 Moving down to 'soon' as not such a priority for now :)

**Comment by kevincrepin** [comment]
Yes I think the group mail can have the same structure. Removed the "topic group" from the design.
